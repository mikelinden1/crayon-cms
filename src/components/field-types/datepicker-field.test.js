import DatePickerField from './datepicker-field';

const props = {
    label: 'Test',
    name: 'test',
    value: 'Wed, 14 Feb 1990 17:49:00 +0000',
    onChange: () => {},
    disabled: false,
    showTime: true,
    timeInterval: 15
};

it('DatePickerField matches snapshot', () => {
    const wrapper = shallow(<DatePickerField {...props} />);
    expect(wrapper).toMatchSnapshot();
});
