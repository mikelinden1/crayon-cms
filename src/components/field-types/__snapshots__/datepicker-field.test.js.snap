// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatePickerField matches snapshot 1`] = `
<DatePicker
  allowSameDay={false}
  className="form-control"
  dateFormat="LLL"
  dateFormatCalendar="MMMM YYYY"
  disabled={false}
  disabledKeyboardNavigation={false}
  dropdownMode="scroll"
  monthsShown={1}
  onBlur={[Function]}
  onChange={[Function]}
  onChangeRaw={[Function]}
  onClickOutside={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onMonthChange={[Function]}
  onSelect={[Function]}
  onYearChange={[Function]}
  preventOpenOnFocus={false}
  selected={"1990-02-14T17:49:00.000Z"}
  shouldCloseOnSelect={true}
  showTimeSelect={true}
  timeCaption="Time"
  timeIntervals={15}
  withPortal={false}
/>
`;
